src/pytest_docker/plugin.py

```diff
@@ -10,6 +10,13 @@
 import pytest
 
 
+@pytest.fixture
+def container_scope_fixture(request):
+    return request.config.getoption("--container-scope")
+
+def containers_scope(fixture_name, config):
+    return config.getoption("--container-scope", "module")
```

Comment:
Should this be `return config.getoption("--container-scope", "session")` as well?